src/Validation/Validator.php

```diff
@@ -1941,6 +1943,54 @@ public function regex($field, $regex, $message = null, $when = null)
         ]);
     }
 
+    /**
+     * Gets the required message for a field
+     *
+     * @param string $field Field name
+     * @return string
+     */
+    public function getRequiredMessage($field)
+    {
+        $defaultMessage = static::DEFAULT_REQUIRED_MESSAGE;
+        if ($this->_useI18n) {
+            $defaultMessage = __d('cake', $defaultMessage);
```

Comment:
This will make the po file extractor skip these messages, as it cannot resolve variables or constants.

Reply:
TIL :)